### PR TITLE
Add instructions to reference the artifact

### PR DESCRIPTION
**Pull request by CharlyCst**

### CODE REVIEW

File: pages/32-instructions.md [hunk 1]
```diff
@@ -93,6 +93,11 @@ Different from traditional evaluations, note that the JSys Artifact Evaluation p
 
 Refer to the [Artifact Evaluation page](/artifact_evaluation) for details about the evaluation procedure, expectations, and tips for authors.
 
+### Referencing the Artefact
+
+The recommended method to reference an artifact is using a service like [Zenodo](https://about.zenodo.org/) to obtain a DOI and a persistent link.
```
Comment (romain-jacob) [inline pages/32-instructions.md]: We miss something like "JSys recommends authors to publish their artifacts publicly and link to them in the final version of the manuscripts.

Comment (romain-jacob) [reply pages/32-instructions.md]: Then mention the "caveat:" As links are volatile, JSys recommends publishing artifacts in a data repository (e.g., Zenodo) that offers persistent identifiers for data. 

And clarify that the authors should use that: Authors can then include the Zenodo-provided DOI to link to their artifacts in a persistent matter, even if the artifact is later updated or moved.

(something like that)

**Comment by romain-jacob** [comment]
Hi @CharlyCst , I forgot about those PR. Could we finish and close them?

**Comment by CharlyCst** [comment]
I also forgot about it, my bad!
I'll fix this this week-end.

**Comment by CharlyCst** [comment]
Pushed an updated version with your feedback @romain-jacob

**Comment by romain-jacob** [comment]
Looks good, thanks!

**Pull request merged**